commands/hero.js

```diff
@@ -39,6 +39,7 @@ const Anilist = new anilist();
 
 cmd(['hero'], withUserEffects(async (ctx, user, effects, ...args) => {
     const now = new Date()
+    await user.save()
```

Comment:
That's interesting, but I guess it works.

Reply:
So far this only does and saves the changes on checking those slots. hence why that save is only there and in ->slots. I can move it to with user effects. but this is less saving and we know there is nothing else being edited with the user at this time

Reply:
I think it could be even put into `withUserEffects`